docs/en/pid-data-model.rst

```diff
@@ -0,0 +1,393 @@
+
+.. code-block::
+
+  <SD-JWT>~<Disclosure 1>~<Disclosure 2>~...~<Disclosure N>
+
+See `[draft-terbu-sd-jwt-vc-latest] <https://vcstuff.github.io/draft-terbu-sd-jwt-vc/draft-terbu-sd-jwt-vc.html>`_ and `[SD-JWT] <https://datatracker.ietf.org/doc/html/draft-ietf-oauth-selective-disclosure-jwt-04>`__ for more details. 
+
+
+
+The JOSE header contains the following mandatory parameters:
+
+.. _pid_jose_header:
+
+.. list-table:: 
+  :widths: 20 60 20
+  :header-rows: 1
+
+  * - **Claim**
+    - **Description**
+    - **Reference**
+  * - **typ**
+    - MUST be set to ``vc+sd-jwt`` as defined in `[draft-terbu-sd-jwt-vc-latest] <https://www.ietf.org/archive/id/draft-terbu-sd-jwt-vc-02.html>`__. 
+    - `[RFC7515, Section 4.1.9] <https://datatracker.ietf.org/doc/html/rfc7515#section-4.1.9>`_.
+  * - **alg**
+    - Signature Algorithm. 
+    - `[RFC7515, Section 4.1.1] <https://datatracker.ietf.org/doc/html/rfc7515#section-4.1.1>`_.
+  * - **kid**
+    - Unique identifier of the public key. 
+    - `[RFC7515, Section 4.1.8] <https://datatracker.ietf.org/doc/html/rfc7516.html#section-4.1.8>`_.
+  * - **trust_chain**
+    - JSON array containing the trust chain that proves the trust of the issuer of this JWT. 
+    - `[OIDC-FED, Section 3.2.1] <https://openid.net/specs/openid-connect-federation-1_0.html#name-trust-chain-header-paramete>`_.
+
+The following claims MUST be in the JWT payload and MUST NOT be included in the Disclosures,  i.e. cannot be selectively disclosed. 
+
+.. list-table:: 
+    :widths: 20 60 20
+    :header-rows: 1
+
+    * - **Claim**
+      - **Description**
+      - **Reference**
+    * - **iss**
+      - The PID Provider identifier as URL string (the issuer of this JWT)
+      - `[RFC7519, Section 4.1.1] <https://www.iana.org/go/rfc7519>`_.
+    * - **sub**
+      - Thumbprint of the JWK in the ``cnf`` parameter
+      - `[RFC7519, Section 4.1.2] <https://www.iana.org/go/rfc7519>`_.
+    * - **jti**
+      - Unique Token ID identifier of this JWT. It MUST be a String in *uuid4* format.
+      - `[RFC7519, Section 4.1.7] <https://www.iana.org/go/rfc7519>`_.
+    * - **iat**
+      - UNIX Timestamp with the time of JWT issuance, coded as NumericDate as indicated in :rfc:`7519`. 
+      - `[RFC7519, Section 4.1.6] <https://www.iana.org/go/rfc7519>`_.
+    * - **exp**
+      - UNIX Timestamp with the expiry time of the JWT, coded as NumericDate as indicated in :rfc:`7519`.
+      - `[RFC7519, Section 4.1.4] <https://www.iana.org/go/rfc7519>`_.
+    * - **status**
+      - HTTPS URL where the credential validity status is available
+      - `[SD-JWT-VC. Section 4.2.2.2] <https://vcstuff.github.io/draft-terbu-sd-jwt-vc/draft-terbu-sd-jwt-vc.html#section-4.2.2.2>`_.
+    * - **cnf**
+      - JSON object containing the proof-of-possession key materials
+      - `[RFC7800, Section 3.1] <https://www.iana.org/go/rfc7800>`_.
```

Comment:
can we add this attribute in the non-normative example?

Reply:
I will provide it asap.